_templates/layout.html

```diff
@@ -2,6 +2,12 @@
 
 {% block footer %}
 
+<style>
+    body {
+      background-image: url("_static/img/6073083.jpg");
+    }
+<style>
+
```

Comment:
there is a `custom.css` file in the `_static` folder which is the best place to put css.

Do you want this on every page? or the slideshow only?